### ISSUE TITLE
Normal users added by 'self registration' can archive boards, cards, swimlanes

### ISSUE THREAD

**Issue by csjjpm** [wekan/wekan#3642]
## Issue

Email settings:
- https://github.com/wekan/wekan/wiki/Troubleshooting-Mail

Add these issues to elsewhere:
- SECURITY ISSUES: https://github.com/wekan/wekan/blob/master/SECURITY.md
- UCS: https://github.com/wekan/univention/issues

Other Wekan issues can be added here.

**Server Setup Information**:

* Note: Please anonymize info, and do not add to this public issue any of your Wekan board URLs, passwords, API tokens etc, do you understand?: Yes
* Did you test in newest Wekan?: Yes
* For new Wekan install, did you configure root-url correctly so Wekan cards open correctly https://github.com/wekan/wekan/wiki/Settings ? Yes 
* Wekan version: 5.04.0
* If this is about old version of Wekan, what upgrade problem you have?: N/A
* Operating System: Ubuntu 20.04 server
* Deployment Method(snap/docker/sandstorm/mongodb bundle/source): docker
* Http frontend if any (Caddy, Nginx, Apache, see config examples from Wekan GitHub wiki first): nginx
* Node Version: 12.21.0
* MongoDB Version: 4.4.4
* Wekan only works on newest desktop Firefox/Chromium/Chrome/Edge/Chromium Edge and mobile Chrome. What webbrowser version are you using? 89.0.4389.82

**Problem description**:
- *REQUIRED: Add recorded animated gif about how it works currently, and screenshot mockups how it should work. Use peek to record animgif in Linux https://github.com/phw/peek*
- *Explain steps how to reproduce*
- *In webbrowser, what does show Right Click / Inspect / Console ? Chrome shows more detailed info than Firefox.*
- *If using Snap, what does show command `sudo snap logs wekan.wekan` ? Please anonymize logs.* 
- *If using Docker, what does show command `sudo docker logs wekan-app` ? Please anonymize logs.*
- *If logs are very long, attach them in .zip file*

I have created a user via self-registration and they are able to use all the archive features.  In the 'people' table they are not Admin.  I would expect them to be a 'normal user'.  In the API there is an option for 'worker', 'comment' , 'no comment' but these aren't available via the web GUI.

**Comment by rokokoh** [comment]
It's not the same problem from this ticket #3377 ?

Beceause in my install normal user, created from LDAP, act as Admin. They can archive and duplicate a board.

![normal-user-01](https://user-images.githubusercontent.com/69970181/133790072-1907ecec-dd79-4889-a0da-c6396b30073f.png)
![normal-user-02](https://user-images.githubusercontent.com/69970181/133790076-0cb6f6eb-49a5-450d-af7e-2a53503e4c18.png)

**Comment by xet7** [comment]
@rokokoh 

Do those LDAP users belong to some LDAP Admin group? Those do get Admin access, like is at Univention that uses Docker Wekan.

**Comment by rokokoh** [comment]
Yes, my bad this LDAP user is Admin user in wekan, sorry i'm new. LDAP user who are not admin in wekan can't archive board if they have "Normal" rights on a board.

**Comment by xet7** [comment]
@rokokoh 

Yes, normal users can not archive board. See #3377

**Comment by xet7** [comment]
Admin Panel / Settings, disable self-registration
